doc/riverctl.1.scd

```diff
@@ -407,62 +407,70 @@ The _input_ command can be used to create a configuration rule for an input
 device identified by its _name_.
 The _name_ of an input device consists of its type, its numerical vendor id,
 its numerical product id and finally its self-advertised name, separated by -.
+A _glob_ pattern can be used in place of _name_ to match input devices with less
```

Comment:
`_name_` with be replaced with `_glob_` in this section once everything is worked out in code.